bioimageio/spec/model/v0_4/raw_nodes.py

```diff
@@ -48,6 +46,9 @@
 PreprocessingName = PreprocessingName
 
 FormatVersion = Literal["0.4.0"]  # newest format needs to be last (used in __init__.py)
+WeightsFormat = Literal[
+    "pytorch_state_dict", "torchscript", "keras_hdf5", "tensorflow_js", "tensorflow_saved_model_bundle", "onnx"
```

Comment:
Does the order here matter? Does it determine the weight preferences in core?

Reply:
no, core has its own list (as it also may have a shorter list). It needs a separate update here: https://github.com/bioimage-io/core-bioimage-io-python/blob/1200b5a9d9b51430217c1adea88929f880b4e038/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py#L10